src/components/projecteditor/ProjectEditor.js

```diff
@@ -127,12 +128,24 @@ export default class ProjectEditor extends Component {
                     onProjectSelected={this.onProjectSelectedHandle}
                 />
                 <div className={style.mainWrapper}>
+                    <div className={classnames([style.sideButtonsContainer, style.sideButtonsContainerLeft])}>
+                        <SideButton name="Explorer"
+                            icon={<IconMosaic style={{width: 12, marginLeft: 2}} />}
+                            onClick={() => { 
+                                    toggleFileSystemPanel(); 
+                                    this.onPanesSizeChange();
+                                }
+                            }  
+                        />
+                    </div>
+
                     <div className={style.mainLayout}>
                         <SplitterLayout 
                             primaryIndex={1}
                             secondaryMinSize={0}
                             secondaryInitialSize={280}
-                            onSecondaryPaneSizeChange={() => this.onPanesSizeChange()}>
+                            onSecondaryPaneSizeChange={() => this.onPanesSizeChange()}
+                            customClassName={!displayFileSystemPanel ? "hideFileSystemPanel" : null}>
```

Comment:
Is it possible to hide/show file explorer panel in a same way as TransactionLogPanel and PreviewSidePanel?

Reply:
Unfortunately there is a problem that other components are dependent on `Control` component being mounted. Also there was issue that it's quite heavy component and it would slow down a lot if we mount/unmount it often.

Reply:
Oh, I suspected that. Lets keep it like this right now then.